chibuezeayogu/React-Math-Skills-Game
# #160780 Add numbers component

**What does the PR do?**
- Add `Numbers Component`

**Description of Task to be completed?**
- add numbers component
- write a unit test for numbers component

**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Math-Skills-Game.gitt)
-  cd into `React-Math-Skills-Game` and checkout to this branch `ft-add-numbers-component-160780636`
- run `npm start`
- run `npm test` to ensure all test are passing 

**What are the relevant pivotal tracker stories?**
- [#160744146](https://www.pivotaltracker.com/story/show/160744146)

**Screenshots**
- N/A
